### ISSUE TITLE
avoiding ALE

### ISSUE THREAD

**Issue by RealYukiSan** [quick-lint/quick-lint-js#1209]
does this plugin able to use native built-in lsp client? I prefer using built-in feature if possible

**Comment by strager** [comment]
Are you talking about Neovim? There is instructions on installing quick-lint-js with nvim-lspconfig here: https://quick-lint-js.com/install/neovim/

**Comment by RealYukiSan** [comment]
yep, I want to install this plugin on neovim, but as stated in the title above, I want to avoid installing ALE if possible

I saw it's a requirement on the installation guide: `[ALE - Asynchronous Lint Engine](https://github.com/dense-analysis/ale) v2.1.1 or newer, or version v3.0.0 or newer`

**Comment by strager** [comment]
The docs say "one of":

> Install and configure **one of** the following Neovim plugins:

You don't need to install ALE if you use nvim-lspconfig (or coc.nvim).

I should make this wording better somehow. Do you have suggestions?

**Comment by RealYukiSan** [comment]
Ah, it's my bad. I'm sorry for my ignorance
I'm not aware of that, and for suggestion, maybe highlight the word using bold and italic would improve typography for some poor-english kid like me XD
